cli/commands/search.py

```diff
@@ -0,0 +1,166 @@
+import rich
+import json
+import chevron
+from .. import ContentType, term_directory
+from . import (
+    prompt_user_to_choose_a_content_type,
+    session,
+    rank_client,
+    prompt_user_to_choose_a_remote_index,
+    prompt_user_to_choose_a_local_query,
+)
+import typer
+from typing import Optional
+from datetime import datetime
+from ..services import elasticsearch
+
+
+app = typer.Typer(
+    name="search",
+    help="Run searches against candidate indices with candidate queries",
+)
+
+
+@app.callback(invoke_without_command=True)
+def main(
+    ctx: typer.Context,
+    search_terms: Optional[str] = typer.Option(
+        None,
+        help="The search terms to use",
+        prompt="What are you looking for?",
+    ),
+    index: Optional[str] = typer.Option(
+        None,
+        help="The index to search in",
+        callback=prompt_user_to_choose_a_remote_index,
+    ),
+    query_path: Optional[str] = typer.Option(
+        None,
+        help="The query to run",
+        callback=prompt_user_to_choose_a_local_query,
+    ),
+    n: Optional[int] = typer.Option(
+        10,
+        help="The number of results to return",
+    ),
+):
```

Comment:
This function is quite confusing - it appears to be hardcoded to refer to works search but it still uses the `ContentType` abstraction. Given the coupling of the entire tool to works and images maybe we could just provide separate functions for each?